### PR TITLE
adding PreferenceManager.getDefaultSharedPreferencesName to default

### PR DESCRIPTION
**Pull request by kibotu**
## Summary

Adding context [PreferenceManager.getDefaultSharedPreferencesName](https://developer.android.com/reference/android/preference/PreferenceManager.html#getDefaultSharedPreferences(android.content.Context)) by default. 

## Changelog

Adding context default shared preference by default.

### CODE REVIEW

**Comment by kibotu** [comment]
done @facebook-github-bot

File: android/src/main/java/com/facebook/flipper/plugins/sharedpreferences/SharedPreferencesFlipperPlugin.java [hunk 1]
```diff
@@ -128,10 +130,18 @@ public boolean accept(File dir, String name) {
...
+    descriptors.add(new SharedPreferencesDescriptor(getDefaultSharedPreferencesName(context), MODE_PRIVATE));
+          
     return descriptors;
   }
 
+  private static String getDefaultSharedPreferencesName(Context context) {
+      return Build.VERSION.SDK_INT >= Build.VERSION_CODES.N
+              ? PreferenceManager.getDefaultSharedPreferencesName(context)
+              : context.getPackageName() + "_preferences";
+
```
Comment (jknoxville) [inline android/src/main/java/com/facebook/flipper/plugins/sharedpreferences/SharedPreferencesFlipperPlugin.java]: I think you're missing a curly brace here to end the method, which is why the CI builds are failing.

**Comment by jknoxville** [comment]
Thanks for the submission, sorry for the delay in getting to it. Looks good to me but there seems to be a missing curly brace: https://github.com/facebook/flipper/pull/454/files#r292027293

If you can fix that up I'll get it merged.

**Comment by kibotu** [comment]
oops sorry about that. also thanks for considering the PR :)

**Pull request closed without merging**